src/Toolkit/Toolkit.UI.Controls/ScaleLine/ScaleLine.Windows.cs

```diff
@@ -141,6 +151,50 @@ private void SetVisibility(bool isVisible)
         {
             Visibility = isVisible ? Visibility.Visible : Visibility.Collapsed;
         }
+#if WPF
+        private class ScaleLineAutomationPeer : FrameworkElementAutomationPeer, IValueProvider
+        {
+            private readonly ScaleLine _owner;
+
+            public ScaleLineAutomationPeer(ScaleLine owner) : base(owner)
+            {
+                _owner = owner;
+            }
+
+            protected override string GetLocalizedControlTypeCore() => Properties.Resources.GetString("ScaleLineAutomationTypeName");
+
+            protected override AutomationControlType GetAutomationControlTypeCore()
+            {
+                return AutomationControlType.Text;
+            }
+
+
+            protected override AutomationLiveSetting GetLiveSettingCore() => AutomationLiveSetting.Polite;
+
+            public override object GetPattern(PatternInterface patternInterface)
+            {
+                if (patternInterface == PatternInterface.Value)
+                {
+                    return this;
+                }
+                return base.GetPattern(patternInterface);
+            }
+
+            public string Value
+            {
+                get
+                {
+                    return $"1 to {GetRoundedValue(_owner.MapScale).ToString()}";
```

Comment:
This should be localized. You'd use string.format for this. Ie
return `String.Format(localizedstring, GetRoundedValue(_owner.MapScale))` where `localizedstring` would be `"1 to {0}"`.